vmm/src/cpu.rs

```diff
@@ -673,6 +674,10 @@ impl CpuManager {
         #[cfg(not(feature = "tdx"))]
         let dynamic = true;
 
+        // Set the barrier as vcpu booting + 1 for the main thread signalling
+        // the pause
+        let vcpus_pause_sync = Some(Arc::new(Barrier::new((config.boot_vcpus + 1).into())));
```

Comment:
I don't think it would work with cpu hotplug, as the number of running threads for vcpu are changing.

Reply:
> I don't think it would work with cpu hotplug, as the number of running threads for vcpu are changing.

Oh yes you are right, I'll fix this.
Many thx.